sips/sip-030/sip-030-wallet-interface.md

```diff
@@ -0,0 +1,528 @@
+# Preamble
+
+SIP Number: `To be assigned`
+
+Title: Integration of a Modern Stacks Wallet Interface Standard
+
+Authors: [janniks](https://github.com/janniks), [kyranjamie](https://github.com/kyranjamie), [aryzing](https://github.com/aryzing), [m-aboelenein](https://github.com/m-aboelenein)
+
+Type: Standard
+
+Status: Draft
+
+Created: 10 October 2023
+
+License: BSD 2-Clause
+
+# OPEN QUESTIONS
+
+<!-- todo: remove section before merge -->
+
+- [ ] Should a global single `window.` object be used, or should provider discovery be handled complely differently?
+- [ ] Rename `stx_transferFt` to include SIP010 somewhere?
+- [ ] Should listeners have params? (e.g. txid for tx mined event)
+
+# Abstract
+
+This SIP proposes the integration of the standard into the Stacks blockchain's "Connect" and "Auth" system.
```

Comment:
This is a bit confusing: "...integration of the standard into the Stacks..."
in this context, what is *the standard*?